packages/devtools_app/lib/src/network/network_controller.dart

```diff
@@ -343,26 +347,45 @@ class NetworkController
             final caseInsensitiveSubstring = substring.toLowerCase();
             final matchesUri =
                 r.uri.toLowerCase().contains(caseInsensitiveSubstring);
-            if (matchesUri) return true;
+            if (matchesUri) {
```

Comment:
these four blocks all look the same.
Maybe a helper that takes `bool checkMatch(r, textToMatch, caseInsensitiveSubstring)`

Reply:
alternately just run
filteredData.value.forEach(checkForError)
after building up the filteredData. That seems a bit easier to get all the cases right.

Reply:
done